changelog.txt

```diff
@@ -1,3 +1,7 @@
+0.88.0
+ * BREAKING CHANGE: DynamicRealm.executeTransaction() now directly throws any RuntimeException instead of wrapping it in a RealmException (#1682).
+ * BREAKING CHANGE: DynamicRealm.executeTransaction() throws IllegalArgumentException if the Transaction object is null.
```

Comment:
now throws IllegalArgumentException instead of silently accepting a null Transaction object.